bench/wtperf/wtperf.c

```diff
@@ -762,6 +762,90 @@ worker(void *arg)
 					value_buf[0] = 'a';
 				if (opts->random_value)
 					randomize_value(thread, value_buf);
+
+				/*
+				 * Distribute the modifications across
+				 * whole document
+				 */
+				if (opts->distribute_modify)
+				{
+					int nentries = 16;
+					size_t iter;
+					size_t total_modify_size;
+					size_t modify_size;
+					size_t modify_offset;
+
```

Comment:
I was initially worried that this code didn't handle the case of small values of `value_sz` (like values less than 10 or 16).  I see that `modify_size` will be zero unless `value_sz >= 160`.  Maybe a comment to that effect: "We don't need to worry about small value sizes (less than 10 * nentries), as modify_size will end up zero, and we won't do any copying at all."

Reply:
Style: All local variables should be declared at the top of the function. We never initialize local variables in the declaration.

Reply:
Corrected the coding style in the new approach and also added a comment to explain for the sizes less than 160